tests/migration/migration.go

```diff
@@ -269,12 +271,22 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 		})
 		Context("[Serial] with bandwidth limitations", Serial, func() {
 
-			var repeatedlyMigrateWithBandwidthLimitation = func(vmi *v1.VirtualMachineInstance, bandwidth string, repeat int) time.Duration {
+			updateMigrationPolicyBandwidth := func(migrationPolicy *v1alpha1.MigrationPolicy, bandwidth resource.Quantity) {
+				migrationPolicy, err = virtClient.MigrationPolicy().Get(context.Background(), migrationPolicy.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				if policyBandwidth := migrationPolicy.Spec.BandwidthPerMigration; policyBandwidth != nil && policyBandwidth.Cmp(bandwidth) == 0 {
+					return
+				}
+
+				migrationPolicy.Spec.BandwidthPerMigration = &bandwidth
+				migrationPolicy, err = virtClient.MigrationPolicy().Update(context.Background(), migrationPolicy, metav1.UpdateOptions{})
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
I think we should use Patch here with `patch` pkg

Reply:
I've made the change, but hesitant on this also TBH.

In this specific case, where the migration policy is handled by this test only, how does a patch even help? As far as I'm concerned, the value of using a patch is in cases where the update could fail due to concurrent changes to the object, but in this case it can't really happen. An update call is easier to write and read, so we shouldn't blindly use patches but rather only where updates make sense.

Reply:
Yeah, I understand. That's why I said "should" and not "have to" :) 
Free to switch back if you prefer